packages/core/test/financial-templates/funding-rate-store/FundingRateStore.js

```diff
@@ -74,12 +74,328 @@ contract("FundingRateStore", function(accounts) {
   });
 
   it("Liveness check", async function() {
-    assert(await didContractThrow(FundingRateStore.new(0, finder.address, timer.address)));
+    assert(await didContractThrow(FundingRateStore.new(0, finder.address, timer.address, { rawValue: "0" })));
   });
 
-  it("Initial Funding Rate and Propose Time of 0", async function() {
-    assert.equal((await fundingRateStore.getFundingRateForContract(mockPerpetual.address)).rawValue.toString(), "0");
-    assert.equal((await fundingRateStore.getFundingRateForContract(mockPerpetual.address)).toString(), "0");
+  it("Initial Funding Rate, Reward Rate, and Propose Time of 0", async function() {
+    const getFundingRate = fundingRateStore.getFundingRateForContract;
```

Comment:
These methods have to be called via the `.call()` pattern now because they are not `view` methods anymore becuse they all attempt to publish any pending proposals